docs/content/Breadcrumbs.md

```diff
@@ -27,6 +27,8 @@ This ensures that the NavLink gets `activeClassName='selected'`
 
 ## System props
 
+**Note: System props for components other than `Box` are deprecated and will be removed soon.**
```

Comment:
This will be easy to find and replace across the codebase if we want to teak the message a bit.